onnxruntime/core/framework/cloud_invoker.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma once
+
+#ifdef USE_CLOUD
+#include <memory>
+#include "gsl/gsl"
+#include "core/framework/tensor.h"
+#include "core/common/inlined_containers.h"
+
+namespace onnxruntime {
+
+using CloudEndPointConfig = std::unordered_map<std::string, std::string>;
+using TensorPtr = std::unique_ptr<onnxruntime::Tensor>;
+using TensorPtrArray = onnxruntime::InlinedVector<TensorPtr>;
+using ConstTensorPtrArray = gsl::span<onnxruntime::Tensor* const>;
```

Comment:
TensorPtr is a unique_ptr before, but a regular ptr here. Confusing. Can rename the previous one to may be TensorUniquePtr?

Reply:
line:
```
using ConstTensorPtrArray = gsl::span<onnxruntime::Tensor* const>
```
removed.